odh/base/monitoring/overrides/prometheus-operator/base/prometheus-rules.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: odh-monitoring
+spec:
+  groups:
+    - name: Operate First Availability
+      rules:
+        - alert: Prometheus Service Down
+          expr: up{service="prometheus-operated"} == 0
+          for: 5m
```

Comment:
Is `5m` just a safe value for now? I think we should be able to react/alert faster.

Reply:
Generally network blips mean things less than 5 minutes are fairly noisy. I think it depends on the MOC cluster though :-)

Reply:
Yeah, I wasn't sure exactly the right time interval suitable for the MOC cluster so as of now set it to `5m`. We can always revisit and modify this once we actually start observing the alerts, wdyt?

Reply:
I'm good with that